bin/generate

```diff
@@ -47,7 +47,7 @@ parser.on('--verify', 'Verify all exercises') do
       fail VerificationError unless current_code == generated_code
     end
     rescue VerificationError => e
-      $stderr.puts e.message % {exercise:}
+      stderr.puts e.message % {exercise:}
```

Comment:
```suggestion
      STDERR.puts e.message % {exercise:}
```

My mistake, not sure that this is under test, and so I missed the case of this constant.